kubernetes/kserve/examples/gpt_fast/sample_text.json

```diff
@@ -0,0 +1,11 @@
+{
+  "instances": [
+    {
+      "data":
+      {
+        "prompt": "The capital of France",
+        "max_new_tokens": 50
+      }
+    }
+  ]
+}
```

Comment:
We should move to use the generate endpoint from open inference protocol once it is out

Reply:
Noted. If there's an example please point to it so we can update all our examples and tests